aqua_contract/migrate.sh

```
@@ -1,36 +1,164 @@
+
+# Load environment variables from file if it exists
+ENV_FILE=".env.${PROFILE}"
 if [ -f "$ENV_FILE" ]; then
-  echo "Loading environment variables from $ENV_FILE..."
-  export $(grep -v '^#' "$ENV_FILE" | xargs)
-else
-  echo "Environment file $ENV_FILE not found!"
-  exit 1
+    echo -e "${BLUE}Loading environment variables from $ENV_FILE...${NC}"
+    set -a  # automatically export all variables
+    source "$ENV_FILE"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)